internal/catalog/testdata/tasks/go-crane-image/go-crane-image.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
     app.kubernetes.io/version: "0.5.0"
   annotations:
     tekton.dev/pipelines.minVersion: "0.50.0"
+    tekton.dev/source: "https://fake.host/repo/resources.tar.gz"
```

Comment:
Any reason it appears there instead of one line above ? (aka just above `tekton.dev/pipelines.minVersion: …`)